submissions/elbeqqal94/solution.js

```diff
@@ -0,0 +1,83 @@
+const fs = require('fs');
+
+const INPUT_FILE = '../../input.txt';
+const OUTPUT_FILE = '../../output.txt';
+
+fs.readFile(INPUT_FILE, 'utf8', (err, data) => {
```

Comment:
This loads the whole content of the fata in memory. Please update

Reply:
Hi @aboullaite I updated my PR, Can you please check again
Thanks!